src/components/PaginationItem/PaginationItem.tsx

```diff
@@ -0,0 +1,24 @@
+interface Props {
+  page: number;
+  currentPage: number;
+  setCurrentPage: (page: number) => void;
+}
+
+export const PaginationItem: React.FC<Props> = ({
+  page,
+  currentPage,
+  setCurrentPage,
+}) => {
+  return (
+    <li className={`page-item ${currentPage === page ? 'active' : ''}`}>
+      <a
+        data-cy="pageLink"
+        className="page-link"
+        onClick={() => setCurrentPage(page)}
```

Comment:
To ensure the page doesn't jump to the top when the link is clicked, you should add `event.preventDefault()` inside the `onClick` handler.